ares/ng/lspc/lspc.cpp

```diff
@@ -13,10 +13,10 @@ auto LSPC::load(Node::Object parent) -> void {
 
   screen = node->append<Node::Video::Screen>("Screen", 320, 256);
   screen->colors(1 << 17, {&LSPC::color, this});
-  screen->setSize(320, 256);
-  screen->setScale(2.0, 2.0);
+  screen->setSize(304, 240);
```

Comment:
Note that for neo geo, some games do intend to use the entire 320x240 framebuffer, while others leave the outer pixels blank

We should use 320x240 here, and as the viewport, rather than clamping to 304.

Reply:
I'm pretty confused about this still. These are the only two combinations I could find that did not force the image into a non-integer scale

  screen = node->append<Node::Video::Screen>("Screen", 320, 256);
  screen->colors(1 << 17, {&LSPC::color, this});
  screen->setSize(320, 256);
  screen->setScale(1.0, 1.0);
  screen->setAspect(1.0, 1.0);
  screen->setViewport(0, 0, 320, 256);
![mslug 2023-01-27 16-46-53](https://user-images.githubusercontent.com/57089001/215221107-a68de854-14a5-4a04-989f-945fe05239e1.png)

  screen = node->append<Node::Video::Screen>("Screen", 320, 256);
  screen->colors(1 << 17, {&LSPC::color, this});
  screen->setSize(304, 240);
  screen->setScale(1.0, 1.0);
  screen->setAspect(1.0, 1.0);
  screen->setViewport(8, 8, 304, 240);
![mslug 2023-01-27 16-47-47](https://user-images.githubusercontent.com/57089001/215221116-eabd6edd-2019-4bb1-9145-ebe52a8eb70f.png)

This is the 1.05ish scaling that the current code does that I was trying to avoid
![gg1](https://user-images.githubusercontent.com/57089001/215226260-afd609ef-bfdf-4d05-ab06-7a90015a3f49.png)

Reply:
Note that many games draw in the overscan area just fine and some even put things there that would otherwise look cut off. This is Blue's Journey in 320 vs 304:
![bjourney 2023-01-27 17-54-17](https://user-images.githubusercontent.com/57089001/215227220-79cc6ae0-8e7e-4493-9e39-fca9790b9c43.png)
![bjourney 2023-01-27 17-55-08](https://user-images.githubusercontent.com/57089001/215227229-dbf56932-0121-4837-98b7-1fd9fa2c8bd2.png)

Reply:
In this case I believe we should crop the outer pixels based on the overscan setting, so setSize and setViewport would change based on the overscan flag.